requirements/DATA_STORE_AURORA_0007/review-of-design.md

```diff
@@ -0,0 +1,9 @@
+# Review of design DATA_STORE_AURORA_0007
```

Comment:
So, this does not require a model, right?

Reply:
It does not

Reply:
Yes, validation method is D, description was discussed on teams.